packages/forms/src/elements/MultiThumbRange.spec.tsx

```diff
@@ -98,7 +206,10 @@ describe('MultiThumbRange', () => {
     it('updates max value if slider is clicked near max thumb', () => {
       const onChangeSpy = jest.fn();
       const { getByTestId } = render(
-        <MultiThumbRange minValue={15} maxValue={75} step={5} onChange={onChangeSpy} />
+        <Field>
+          <Label>MultiThumbRange</Label>
```

Comment:
Do all of these existing tests require a `Label`?

Reply:
Jumping from the comment about breaking changes -> I will update these tests so that multi-thumb range usages do not require a `Label`. New commits coming shortly 👍 .